packages/@n8n/benchmark/scripts/runForN8nSetup.mjs

```diff
@@ -36,6 +36,14 @@ async function main() {
 	const vus = argv.vus;
 	const duration = argv.duration;
 
+	const hasN8nLicense = !!n8nLicenseCert || !!n8nLicenseActivationKey;
+	if (n8nSetupToUse === 'scaling' && !hasN8nLicense) {
+		console.error(
+			'n8n license is required to run the scaling setup. Please provide N8N_LICENSE_CERT or N8N_LICENSE_ACTIVATION_KEY',
```

Comment:
Strictly speaking scaling mode doesn't require a license, only multi-main does.

Reply:
Isn't scaling mode == multi-main?

Reply:
Scaling mode is the Bull-based setup, which comes in single-main (free) and multi-main (licensed) flavors.

Reply:
Right, and queue mode is the legacy name for scaling mode? I think we should clarify the naming.